stock_picking_type_shipping_policy/models/stock_picking_type.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+SHIPPING_POLICY_SELECTION = [
+    ("procurement_group", "Take from procurement group"),
+    ("force_as_soon_as_possible", "Force to as soon as possible"),
+    ("force_all_products_ready", "Force to when all products are ready"),
+]
+
+
+class StockPickingType(models.Model):
+
+    _inherit = "stock.picking.type"
+
+    @api.model
+    def _default_shipping_policy(self):
+        return "procurement_group"
+
+    shipping_policy = fields.Selection(
+        SHIPPING_POLICY_SELECTION,
+        default=lambda r: r._default_shipping_policy(),
```

Comment:
I think you don't need to split selection options to a var. You could delegate them to a method instead.
You don't need the default method but if you move options to a method then it's better if you keep it.